src/api/loader.coffee

```diff
@@ -41,22 +41,27 @@ getResponseDataByEnv = (isLocal, eventData, response) ->
 
 
 handleAPIEvent = (apiEventChannel, baseUrl, setting, eventData = {}) ->
+
   isLocal = setting.loadLocally
   # simulate server delay
   delay = if isLocal then 200 else 0
 
   apiSetting = getAjaxSettingsByEnv(isLocal, baseUrl, setting, eventData)
+  if apiSetting.method is 'GET'
```

Comment:
Should this really only check for GET requests?  i guess we could do it based on if it's the same method and same body as well, but i guess that could get a little complex